src/pandablocks_ioc/ioc.py

```diff
@@ -175,6 +177,45 @@ def create_softioc(
             asyncio.run_coroutine_threadsafe(client.close(), dispatcher.loop).result()
 
 
+async def get_panda_versions(
+    client: AsyncioClient,
+) -> dict[EpicsName, str]:
+    """Function that gets version information from the PandA using the IDN command
```

Comment:
Just thought it might be good to link:

https://pandablocks-server.readthedocs.io/en/latest/commands.html#system-commands

here too to explain why we need the parsing logic.